### PR TITLE
Add sku defenition for the cisco Nexsus C3164PQ

### PR DESCRIPTION
**Pull request by tannoa2**

### CODE REVIEW

**Comment by tannoa2** [comment]
@RackHD/corecommitters @jlongever @amymullins @uppalk1 @keedya @zyoung51

**Comment by tannoa2** [comment]
For your convenience I pasted below the catalog of the switch:

```
onrack@ora:/$ curl http://localhost:8080/api/common/nodes/"573e3930eace2fc40895d5f7"/catalogs | python -mjson.tool
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1950  100  1950    0     0   192k      0 --:--:-- --:--:-- --:--:--  211k
[
    {
        "createdAt": "2016-05-19T22:07:47.064Z",
        "data": {
            "bios_cmpl_time": "09/10/2014",
            "bios_ver_str": "08.06",
            "bootflash_size": "51496280",
            "chassis_id": "Nexus3000 C3164PQ Chassis",
            "cpu_name": "Intel(R) Xeon(R) CPU E5-2420",
            "header_str": "Cisco Nexus Operating System (NX-OS) Software\nTAC support: http://www.cisco.com/tac\nCopyrigh                                                        t (C) 2002-2015, Cisco and/or its affiliates.\nAll rights reserved.\nThe copyrights to certain works contained in this so                                                        ftware are\nowned by other third parties and used and distributed under their own\nlicenses, such as open source.  This s                                                        oftware is provided \"as is,\" and unless\notherwise stated, there is no warranty, express or implied, including but not\                                                        nlimited to warranties of merchantability and fitness for a particular purpose.\nCertain components of this software are                                                         licensed under\nthe GNU General Public License (GPL) version 2.0 or \nGNU General Public License (GPL) version 3.0  or th                                                        e GNU\nLesser General Public License (LGPL) Version 2.1 or \nLesser General Public License (LGPL) Version 2.0. \nA copy o                                                        f each such license is available at\nhttp://www.opensource.org/licenses/gpl-2.0.php and\nhttp://opensource.org/licenses/g                                                        pl-3.0.html and\nhttp://www.opensource.org/licenses/lgpl-2.1.php and\nhttp://www.gnu.org/licenses/old-licenses/library.tx                                                        t.",
            "host_name": "switch",
            "kern_uptm_days": "0",
            "kern_uptm_hrs": "0",
            "kern_uptm_mins": "2",
            "kern_uptm_secs": "19",
            "kick_cmpl_time": "1/26/2015 11:00:00",
            "kick_file_name": "bootflash:///n9000-dk9.6.1.2.I3.3a.bin",
            "kick_tmstmp": "01/26/2015 19:45:44",
            "kickstart_ver_str": "6.1(2)I3(3a)",
            "manufacturer": "Cisco Systems, Inc.",
            "mem_type": "kB",
            "memory": "16401436",
            "proc_board_id": "SAL2011M59M",
            "rr_ctime": "Fri May 20 14:06:08 2016",
            "rr_reason": "Reset Requested by CLI command reload",
            "rr_service": null,
            "rr_sys_ver": "6.1(2)I3(3a)",
            "rr_usecs": "244996"
        },
        "id": "573e3933960adbc008d51a8f",
        "node": "573e3930eace2fc40895d5f7",
        "source": "version",
        "updatedAt": "2016-05-19T22:07:47.064Z"
    }
]
Z"
    }
]
```

**Comment by zyoung51** [comment]
+1

File: cisco-nexus-C3164PQ/config.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Cisco Nexus C3164PQ",
+    "rules": [
+        {
+            "path": "version.chassis_id",
+            "regex": "Nexus3000 C3164PQ Chassis"
```
Comment (yyscamper) [inline cisco-nexus-C3164PQ/config.json:6]: Minor suggestion.
This more like to be `equals` or `contains` than `regex`. anyway the `regex` will still work.

**Comment by yyscamper** [comment]
:+1:

**Comment by anhou** [comment]
:+1:

**Pull request merged**